agent/action/arbitrage_result.py

```
@@ -5,11 +5,222 @@
+
+def _load_rescue_cfg(context):
+    """从 _RESCUE_NODE.attach 覆盖救援可调参(缺项保留 py 默认)。run 起始调一次,幂等。"""
+    try:
+        node = context.get_node_object(_RESCUE_NODE)
+        attach = getattr(node, "attach", None) if node else None
+        if not attach:
+            return
+        for k in _RESCUE_CFG:
+            if k in attach:
+                _RESCUE_CFG[k] = type(_RESCUE_CFG[k])(attach[k])
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)